backend/domain/domainmodels.js

```diff
@@ -0,0 +1,134 @@
+
+
+
+
+function Room (){
+
+        this.roomName = this.setRoomName(); //@@@@@@@ -some function to initialise a roomID
+
```

Comment:
Do we need this whitespace between each line setting a property

Reply:
I have fixed the white space inconsistencies